src/server/Document/component.test.jsx

```diff
@@ -35,8 +35,9 @@ describe('Document Component', () => {
   );
   const styleTagsAmp = <style amp-custom="">{'html { color: red; }'}</style>;
 
-  const shallowDocument = ({ service, isAmp }) =>
-    shallowRender(
+  const testDocumentComponent = ({ service, isAmp, testName }) =>
```

Comment:
I would change this function. something like this
```
  const document = ({ service, isAmp }) =>
    <DocumentComponent
        assets={assets}
        assetOrigins={assetOrigins}
        app={'<h1>App!</h1>'}
        data={{ ...data }}
        helmet={helmet}
        styleTags={isAmp ? styleTagsAmp : styleTags}
        service={service}
        isAmp={isAmp}
      />
```

then for your tests you can

```
shouldMatchSnapshot('should render correctly', document('news',false);
```